lib/libmbedtls/include/mbedtls_config_uta.h

```diff
@@ -15,6 +15,7 @@
 #define MBEDTLS_CIPHER_C
 #define MBEDTLS_DES_C
 #define MBEDTLS_AES_C
+#define MBEDTLS_NIST_KW_C
```

Comment:
Please make this change in a separate commit.

Reply:
Done